shared/word2vec_download.py

```diff
@@ -11,7 +11,7 @@
 from utils import download
 
 
-def word2vec_download(to_path="installed", replace_existing=True) -> str:
+def word2vec_download(to_path="installed", replace_existing=False) -> str:
```

Comment:
does any architecture still included use word2vec anymore? If not, can we just get rid of it completely?

Reply:
Yes, we are using for dbscan clustering and word2vec similarity.